### ISSUE TITLE
More tech: Electricity, Biology, Physics, Bio-Tech, Nuclear Physics, And More!

### ISSUE THREAD

**Issue by CTH999** [Da-Technomancer/Crossroads#226]
**Scope**
A _lot_ of new ideas. There's a reason it's in a seperate doc...
There is, at the top, a "summary" that leads to my favorite features

**Suggestion**
It's [On this Google Doc](https://docs.google.com/document/d/1sOJpVdRi780N0xgxR__8-zrP-iN_06p2/edit?usp=sharing&ouid=118013032969542638389&rtpof=true&sd=true)
1. Chemical manipulations
2. Genetics
3. Cloning
4. Conscisness Transfer, allowing _advanced_ systems to allow hard-core mode respawn
5. Teleportation
6. Nuclear
7. Radiation
8. Fallout
9. Fun!

**Justification**
This mod is all about intricite automation. And yet, so many things can't be automated. And, the electricity is practicly useless. This idea will fix that, as well as improve _everything_! Adding new features, weapons, and more! I could try to make a sales pitch, but I don't know where to begin. Just read it.
More info in a kinda strange and hard to follow (since it's basicly a conversation) [here](https://discord.com/channels/350295689789440000/1017462927495790612)

**Impact**
1. Add new mechanics
2. Alter progression, adding in a sort of "non-magical" version of _all_ magic paths
3. Add entire new sections of the mod that are all, theoreticly, optional
4. Add a bit of realisim without being too much (your electricity is _way_ simplified, this makes it less simplified, but not too-much less

**Comment by Random-Scientist** [comment]
1. Alchemy exists
2. Witchcraft exists (which has genetics)
3. Witchcraft exists (which has cloning)
4. this and 5 are already sort of covered by the technomancy portal, and adding a respawn mechanic to hardcore mode ruins the point of hardcore mode (you get one life)
5. See ^
6. High-tech stuff doesn’t really fit this mod’s steampunk/magitech style. As for powerful energy generation, beacon harnesses work quite well for beams and a simple redstone setup will have you covered for life as far as heat/rotary goes.
7. Again, Crossroads is a mod about choosing a steampunk/magical path and interesting automation challenges. Making things realistic means you’re making a different mod (which you’re welcome to do, btw) with different goals.

A lot of your suggestions fall in a similar vein to 7. They are perfectly good ideas, but at the end of the day Crossroads is not trying to be another Gregtech

**Comment by CTH999** [comment]
> 1. Alchemy exists
> 2. Witchcraft exists (which has genetics)
> 3. Witchcraft exists (which has cloning)
> 4. this and 5 are already sort of covered by the technomancy portal, and adding a respawn mechanic to hardcore mode ruins the point of hardcore mode (you get one life)
> 5. See ^
> 6. High-tech stuff doesn’t really fit this mod’s steampunk/magitech style. As for powerful energy generation, beacon harnesses work quite well for beams and a simple redstone setup will have you covered for life as far as heat/rotary goes.
> 7. Again, Crossroads is a mod about choosing a steampunk/magical path and interesting automation challenges. Making things realistic means you’re making a different mod (which you’re welcome to do, btw) with different goals.
> 
> A lot of your suggestions fall in a similar vein to 7. They are perfectly good ideas, but at the end of the day Crossroads is not trying to be another Gregtech

1- see bellow
2-3 yes, but biology is sort of a weak witchcraft, to allow acces to some of it's abilities, without needing magic. For those who only have acces to one path
4-5. Fair enough
6- fair enough, to _some extent_. Some things, like the electricity, are way oversimplified. and, in fact, back in the more "steampunk" time, electricity was more complex and harder
7- fair enough. You don't have to use all of them, this is for ideas for some. Use them all, or some, or (hopefully not) none.

Gregtech? I've heard of it, but what do you mean?

My favorites are the Electricity, Pnumatics, and Nuclear
       Besides the Electricity Improvments, I basicly removed all new tech (cybernetics is now semi-magic, and part of witchcraft)

**Comment by Da-Technomancer** [comment]
Not planned. A longer response was sent to the submitter in Discord, but in summary for recordkeeping:
- Insane levels of scope creep
- Overlap with other mods
- Thematic clash with Crossroads
- Several of these suggestions are already in the mod
